spec/controllers/concerns/rate_limit_concern_spec.rb

```diff
@@ -15,6 +15,12 @@ def show
       def update
         render plain: 'Bye'
       end
+
+      private
+
+      def flow_session
+        {}
+      end
```

Comment:
Let's discuss why this is needed. I think it might cause problems for other specs.

Reply:
the StepController used in this spec does not include IdvStepConcern which gives access the flow_session method ... i could stub flow_sesssion as an alternative?

Reply:
Stubbing flow_session sounds better to me. Here's what I did in [ssn_controller_spec](https://github.com/18F/identity-idp/blob/53c2b65f22e54bc19c1ac6b655f973aeb0592138/spec/controllers/idv/ssn_controller_spec.rb#L19), with flow_session assigned a hash above.